### ISSUE TITLE
Should embedding normalization be handled manually by the user for callback usage?

### ISSUE THREAD

**Issue by lucas-maes** [rbalestr-lab/stable-pretraining#237]

**Comment by huguesva** [comment]
Hi Lucas, what kind of normalization do you have in mind? batch norm for instance ?
